File: docs/examples/teaching/10-local-logic/inline/recipe.js
export function build(root) {
    root.numberTextBox({value: '^quantity', lbl: 'Quantity', live: true});
    root.numberTextBox({value: '^price', lbl: 'Price', live: true});
    root.p('==quantity * price', {quantity: '^quantity', price: '^price'});
}

File: docs/examples/teaching/10-local-logic/inline/recipe.py
from gramlot.page import WebPage


class Page(WebPage):
    def main(self, root):
        root.numberTextBox(value="^quantity", lbl="Quantity", live=True)
        root.numberTextBox(value="^price", lbl="Price", live=True)
        root.p("==quantity * price", quantity="^quantity", price="^price")
